spec/Section 3 -- Type System.md

```diff
@@ -1815,8 +1923,13 @@ by a validator, executor, or client tool such as a code generator.
 GraphQL implementations should provide the `@skip` and `@include` directives.
 
 GraphQL implementations that support the type system definition language must
-provide the `@deprecated` directive if representing deprecated portions of
-the schema.
+provide:
+
+- the `@deprecated` directive if representing deprecated portions of the
+  schema;
+- the `@oneOf` directive if representing types that require exactly one field
```

Comment:
Technically, as written, this clause would immediately make all GraphQL schemas that have any exclusive input types Not Spec Compliant, since they "represent types that require exactly one field" but don't provide `@oneOf`.

I assume it doesn't matter in practice? But nominally it breaks the migration path.